webapp/tests/forms/steps/lotse_multistep_flow_steps/test_steuerminderungen_steps.py

```diff
@@ -205,22 +205,42 @@ def test_do_not_skip_if_separated(self):
         assert redirection_info[0] is None
         assert redirection_info[1] is None
 
-    def test_skip_if_widowed_zusammenveranlagung(self):
+    def test_do_not_skip_if_widowed_longer(self):
```

Comment:
Because I probably will not understand this in a week :) maybe we could name that `test_do_not_skip_if_widowed_longer_than_veranlagungszeitraum`?